docs/schemas/GainCal.yml

```diff
@@ -50,7 +50,7 @@ inputs:
     default: false
     type: bool
     doc: >-
-      Apply the beam model (at the phase center) to the visibilities in the model column. If this option is true, all options from :ref:`ApplyBeam` are valid as well (except .invert, since the model data will always be corrupted for the beam) `.`
+      Apply the beam model (at the phase center) to the visibilities in the model column. If this option is true, all options from `ApplyBeam <ApplyBeam.html>`__ are valid as well (except .invert, since the model data will always be corrupted for the beam) `.`
```

Comment:
I didn't know the syntax with trailing `__`, is that yaml? If it works, ok with me.

Reply:
it is to define that the href is anonymous. Else it will give weird warnings it has similar hrefs